modules/vuetify/index.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path')
+
+module.exports = function nuxtVuetify (options) {
+  // Add assets
+  this.options.head.link.push({
+    rel: 'stylesheet',
+    type: 'text/css',
+    href: '//fonts.googleapis.com/css?family=Material+Icons'
```

Comment:
Is this recommended by vuetify? That would be nice if we have an option to disable it.